Kotlin/sort/QuickSort/src/QuickSort.kt

```diff
@@ -0,0 +1,71 @@
+//QuickSort Algortihm works on Divide and Conquer Algorithm. It creates two empty arrays to hold elements less than the pivot value and elements greater than the pivot value, and then recursively sort the sub arrays. There are two basic operations in the algorithm, swapping items in place and partitioning a section of the array.
+
+//Here QuickSort is implemented in Kotlin using recursive approach
+
+//Importing the Java.util package, it is needed because we are using the scanner object.
```

Comment:
remove such redundant comments 
- Add Multi-Line Comments 
- Add a beginner-friendly description at the top